blocktopmenu.php

```diff
@@ -802,7 +802,7 @@ protected function getCMSCategories($recursive = false, $parent = 1, $id_lang =
 
 	}
 
-	protected function getCMSPages($id_cms_category, $id_shop = false, $id_lang = false, $id_shop = false)
+	protected function getCMSPages($id_cms_category, $id_shop = false, $id_lang = false)
```

Comment:
Hi, thank you for your contribution, but you may also change call at line 693.

Best regards.